bsc-plugin/src/lib/rooibos/MockUtil.ts

```diff
@@ -186,4 +186,41 @@ export class MockUtil {
         }
     }
 
+    public addRuntimeGlobalFunctionMocks(file: BrsFile, astEditor: Editor) {
+        file.ast.walk(createVisitor({
+            FunctionStatement: (statement, parent, owner, key) => {
+                if (!file.pkgPath.includes('rooibos/') && !file.pkgPath.endsWith('spec.brs')) {
```

Comment:
Need to make this a bit more config driven.

Reply:
agree, this is also slow. we should pull that into the top of the function, instead of in this hot walk matcher.

Reply:
Should no longer be needed as it used the same global detection logic as the `expectCalled` logic.